edxval/admin.py

```diff
@@ -30,6 +33,13 @@ class VideoAdmin(admin.ModelAdmin):  # pylint: disable=C0111
     admin_order_field = 'edx_video_id'
     inlines = [CourseVideoInline, EncodedVideoInline]
 
+
+class VideoImageAdmin(admin.ModelAdmin):
+    model = VideoImage
+    verbose_name = "Video Image"
+    verbose_name_plural = "Video Images"
+
 admin.site.register(Profile, ProfileAdmin)
 admin.site.register(Video, VideoAdmin)
 admin.site.register(Subtitle)
+admin.site.register(VideoImage, VideoImageAdmin)
```

Comment:
can we also add CourseVideo to the admin in this PR or any other/subsequent PR ?